reltest/repository_test.go

```diff
@@ -155,3 +155,18 @@ func TestRepository_Transaction_runtimerError(t *testing.T) {
 
 	repo.AssertExpectations(t)
 }
+
+func TestRepository_Exec(t *testing.T) {
+	var (
+		repo  = New()
+		query = "UPDATE users SET something = ? WHERE something2 = ?;"
+	)
+
+	repo.mock.On("Exec", context.TODO(), query, []interface{}{3, "sdfds"}).Return(2, nil).Once()
```

Comment:
we should add mock method as well, maybe:

```go
func ExpectExec(query string, args ...interface{}) *Expect
```

Reply:
You mean something like 
`func (r *Repository) ExpectExec(stmt string, args []interface{}, rets []interface{}) *Expect {
`

Reply:
`rets` is return?
No, please see other APi, return will be set by Expect using Return function

Reply:
> `rets` is return?
Yes I used the naming from https://github.com/go-rel/rel/blob/3aebb7b94d7c11957507904acecdb0f3bcbb86d0/reltest/expect.go#L29

> No, please see other APi, return will be set by Expect using Return function-
Pls check d525eb7 if it's what you mean.